plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/TrinoCatalog.java

```diff
@@ -159,6 +160,7 @@ Transaction newCreateOrReplaceTableTransaction(
 
     void updateViewColumnComment(ConnectorSession session, SchemaTableName schemaViewName, String columnName, Optional<String> comment);
 
+    @Nullable
     String defaultTableLocation(ConnectorSession session, SchemaTableName schemaTableName);
```

Comment:
defaultTableLocation should be changed to return `Optional`
(ideally `Optional<Location>` rather than `Optional<String>`)